optuna/_hypervolume/base.py

```diff
@@ -42,7 +42,9 @@ def objective(trial):
             print("Hypervolume of the Pareto solutions is {}.".format(hypervolume))
     """
 
-    def compute(self, solution_set: np.ndarray, reference_point: np.ndarray) -> float:
+    def compute(
```

Comment:
How about adding test cases to validate when `assume_pareto` is True or False?

Reply:
Done!